DOCS/man/options.rst

```diff
@@ -6599,17 +6599,25 @@ them.
         Blend the frame against the background color (``--background``, normally
         black).
     yes
-        Try to create a framebuffer with alpha component. This only makes sense
-        if the video contains alpha information (which is extremely rare) or if
-        you make the background color transparent. May not be supported on all
-        platforms. If alpha framebuffers are unavailable, it silently falls
-        back on a normal framebuffer. Note that if you set the ``--fbo-format``
-        option to a non-default value, a format with alpha must be specified,
-        or this won't work. Whether this really works depends on the windowing
-        system and desktop environment.
+        Render the frame as-is, preserving its alpha component. Blending is
+        done on the mpv window by the windowing system. This only works
+        if a framebuffer with alpha transparency is created (see
+        ``--window-transparent``)
     no
         Ignore alpha component.
 
+``--window-transparent=<auto|yes|no>``
+    Try to create a framebuffer with alpha component. This only makes sense
+    if the video contains alpha information (which is extremely rare) or
+    if you make the background color transparent. May not be supported on
+    all platforms. If alpha framebuffers are unavailable, it silently falls
+    back on a normal framebuffer. Note that if you set the ``--fbo-format``
+    option to a non-default value, a format with alpha must be specified,
+    or this won't work. Whether this really works depends on the windowing
```

Comment:
The sentence about `--fbo-format` should go into the documentation of `--alpha=yes`, since it's relevant only for alpha blending *within* the pipeline, rather than display onto the screen.

Reply:
Like this? I'll admit that I don't full understand the workings of `fbo-format`, even after trying its different values out.